kokoro/windows/build.bat

```diff
@@ -48,27 +49,47 @@ cmake -DRE2_BUILD_TESTING=OFF -DCMAKE_C_COMPILER="C:/Program Files (x86)/Microso
 if %ERRORLEVEL% GEQ 1 exit /b %ERRORLEVEL%
 
 echo "Build glslang... %DATE% %TIME%"
-ninja glslangValidator
+::ninja glslangValidator
 if %ERRORLEVEL% GEQ 1 exit /b %ERRORLEVEL%
 
 echo "Build everything... %DATE% %TIME%"
-ninja
+::ninja
 if %ERRORLEVEL% GEQ 1 exit /b %ERRORLEVEL%
 
 echo "Check Shaderc for copyright notices... %DATE% %TIME%"
-ninja check-copyright
+::ninja check-copyright
 if %ERRORLEVEL% GEQ 1 exit /b %ERRORLEVEL%
 echo "Build Completed %DATE% %TIME%"
 
 :: #########################################
 :: Run the tests.
 :: #########################################
 echo "Running Tests... %DATE% %TIME%"
-ctest -C %BUILD_TYPE% --output-on-failure -j4
+::ctest -C %BUILD_TYPE% --output-on-failure -j4
 echo "Tests Completed %DATE% %TIME%"
 
+pwd
```

Comment:
So I guess I should stop paying attention to this PR for a while..... :-)

Reply:
Yes, please ignore all changes to this PR until we can figure out why we're not able to cleanup and the build is done. Sadly there's no easy way to figure things out except by making a change and looking at the logs.